torch/_dynamo/variables/nn_module.py

```diff
@@ -606,6 +606,18 @@ def gen_source(source, name):
                 [self] + args,
                 kwargs,
             )
+        elif name == "_conv_forward":
+            # inline the torch.nn.modules.conv._conv_forward
+            fn = getattr(module, name).__func__
+            module_attr = getattr(fn, "__module__", "")
+            if module_attr is not None and module_attr.startswith(
+                "torch.nn.modules.conv"
+            ):
```

Comment:
I would suggest to use ```isinstance(mod, torch.nn.modules.conv._ConvNd```, and you can get ```mod``` by ```tx.output.get_submodule(self.module_key)```. If users extends from the builtin ```Conv``` module, they can benefit from this inline. We do see similar patterns from torchbench models.